tests/queries/0_stateless/02475_bson_each_row_format.sh

```diff
@@ -5,6 +5,9 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
+# Disable parallel parsing to avoid mixing output in case of small input block (--min_chunk_bytes_for_parallel_parsing)
+CLICKHOUSE_CLIENT+="--allow_repeated_settings --max_threads 1"
```

Comment:
But parallel parsing should preserve the original order of rows. I think the reason is in parallel output from storages (`parallelize_output_from_storages`) when we had small blocks read in File storage. But `max_threads=1` will also help in this case (but let's change the comment as it's misleading)

Reply:
>when we had small blocks read in File storage

Yep, indeed.

>but let's change the comment as it's misleading

I've adjusted the comment, how does it looks to you?